examples/tutorials/asr_inference_with_ctc_decoder_tutorial.py

```diff
@@ -21,6 +21,13 @@
 #    Please refer to https://pytorch.org/get-started/locally/
 #    for instructions.
 #
+#    To enable running the tutorial notebook in Google Colab, install nightly
+#    torch and torchaudio builds by adding the following code block to the top
+#    of the notebook before running it:
+#    ::
+#      !pip3 uninstall -y torch torchvision torchaudio
+#      !pip3 install --pre torch torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/nightly/cpu
+#
```

Comment:
This information is almost equivalent to what https://pytorch.org/get-started/locally/ says, yet it could become obsolete if release team should decide to change the URL.

Since the new information is very specific to Google Colab, how about raising this as error message when `import torchaudio.prototypes` fails?


```python
try:
    import torchaudio.prototype.ctc_decoder
except ModuleNotFoundError:
    print(message)
    raise
```

Reply:
FYI: this is the full snippet for colab to run ASR inference;

```python
!pip3 uninstall -y torch torchvision torchaudio
!pip3 install --pre torch torchaudio --extra-index-url https://download.pytorch.org/whl/nightly/cpu
!pip3 install sentencepiece
!add-apt-repository -y ppa:savoury1/ffmpeg4
!apt-get -qq install -y ffmpeg
```

Reply:
updated